eclass/qt5-build.eclass

```diff
@@ -67,10 +67,11 @@ QT5_MINOR_VERSION=$(ver_cut 2)
 readonly QT5_MINOR_VERSION
 
 case ${PV} in
-	5.??.9999)
-		# git stable branch
+	5.15.9999)
+		# KDE upstream for 5.15 patches
+		HOMEPAGE+="https://invent.kde.org/qt/qt/"
```

Comment:
HOMEPAGE Needs a space between URIs ;)

Sorry re variable, didn't think of that myself.

Reply:
Crud!  Another thing I've made note in the past, and completely didn't think of now, of course.